Solutions/MailRisk/Package/mainTemplate.json

```diff
@@ -30,9 +30,9 @@
     }
   },
   "variables": {
-    "solutionId": "securepractice.azure-sentinel-solution-mailrisk",
+    "solutionId": "securepractice.microsoft-sentinel-solution-mailrisk",
```

Comment:
Hi @ingin97 we are trying to deploy the solution but it is not reflecting post deployment, can you please confirm why this offer id has been updated, thanks!

Reply:
Hi @v-vdixit, the solution is not yet published, and the offer in the Partner for publishing somehow got mismatched with the publisher ID, we were adviced from a Partner Program Manager that the simplest fix was to change the offer name in the solution id for it to match properly.